tests/model_hub_tests/transformation_tests/test_pa_transformation.py

```diff
@@ -14,17 +14,20 @@
 import os
 import re
 
-def compare_diffs(ov_model: ov.Model,
-                  model_id: str,
-                  use_block_indices_inputs: bool,
-                  use_score_outputs: bool,
-                  allow_cache_rotation: bool):
+def apply_transformation_and_compare_diffs(ov_model: ov.Model,
+                                           model_id: str,
+                                           use_block_indices_inputs: bool,
+                                           use_score_outputs: bool,
+                                           allow_cache_rotation: bool,
+                                           ie_device: str):
```

Comment:
```suggestion
                                           device: str):
```
`ie_` prefix is obsolete